modules/guac/ra_guac_autoscale_public_alb.template.cfn.yaml

```diff
@@ -342,108 +342,96 @@ Resources:
-              - 'ssm:ListAssociations'
-              - 'ssm:ListInstanceAssociations'
-              - 'ssm:PutInventory'
-              - 'ssm:UpdateAssociationStatus'
-              - 'ssm:UpdateInstanceAssociationStatus'
-              - 'ssm:UpdateInstanceInformation'
+              - ssm:DescribeAssociation
+              - ssm:GetDeployablePatchSnapshotForInstance
+              - ssm:GetDocument
+              - ssm:ListAssociations
+              - ssm:ListInstanceAssociations
+              - ssm:PutInventory
+              - ssm:UpdateAssociationStatus
+              - ssm:UpdateInstanceAssociationStatus
+              - ssm:UpdateInstanceInformation
             Effect: Allow
             Resource: '*'
           - Action:
-              - 'ec2messages:AcknowledgeMessage'
-              - 'ec2messages:DeleteMessage'
-              - 'ec2messages:FailMessage'
-              - 'ec2messages:GetEndpoint'
-              - 'ec2messages:GetMessages'
-              - 'ec2messages:SendReply'
+              - ec2messages:AcknowledgeMessage
+              - ec2messages:DeleteMessage
+              - ec2messages:FailMessage
+              - ec2messages:GetEndpoint
+              - ec2messages:GetMessages
+              - ec2messages:SendReply
             Effect: Allow
             Resource: '*'
           - Action:
-              - 'cloudwatch:PutMetricData'
+              - cloudwatch:PutMetricData
             Effect: Allow
             Resource: '*'
           - Action:
-              - 'ec2:DescribeInstanceStatus'
+              - ec2:DescribeInstanceStatus
             Effect: Allow
             Resource: '*'
           - Action:
-              - 'ds:CreateComputer'
-              - 'ds:DescribeDirectories'
+              - ds:CreateComputer
+              - ds:DescribeDirectories
             Effect: Allow
             Resource: '*'
           - Action:
-              - 'logs:CreateLogGroup'
-              - 'logs:CreateLogStream'
-              - 'logs:DescribeLogGroups'
-              - 'logs:DescribeLogStreams'
-              - 'logs:PutLogEvents'
+              - logs:CreateLogGroup
+              - logs:CreateLogStream
+              - logs:DescribeLogGroups
+              - logs:DescribeLogStreams
+              - logs:PutLogEvents
             Effect: Allow
             Resource: '*'
           - Action:
-              - 's3:GetObject'
+              - s3:GetObject
             Effect: Allow
             Resource:
-              - 'arn:aws:s3:::amazon-ssm-*'
-              - 'arn:aws:s3:::amazoncloudwatch-agent/*'
+              - arn:aws:s3:::amazon-ssm-*
+              - arn:aws:s3:::amazoncloudwatch-agent/*
           - Action:
-              - 's3:ListBucket'
+              - s3:ListBucket
             Effect: Allow
-            Resource: 'arn:aws:s3:::amazon-ssm-packages-*'
-          - Action:
-              - 'autoscaling:DescribeAutoScalingInstances'
-            Effect: Allow
-            Resource: '*'
-          - Action:
```

Comment:
This was removed because it was over scoped, correct?

Reply:
correct here and the next one. these actions were remnants of an older approach to handling updates. now everything is using actual health checks properly, and no longer makes these calls.